client/my-sites/stats/stats-email-open-detail/index.jsx

```diff
@@ -335,6 +340,8 @@ const connectComponent = connect(
 			hasValidDate,
 		} = getPeriodWithFallback( ownProps.period, ownProps.date, isValidStartDate, post?.date );
 
+		const maxBars = 0;
```

Comment:
Why is this?

Reply:
Initializing the state. It's part of the state that is passed to the component in line 363.